WooCommerce/WooCommerceTests/ViewModels/CardPresentPayments/PaymentReceiptEmailParameterDeterminerTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+import TestKit
+@testable import WooCommerce
+@testable import Yosemite
+
+final class PaymentReceiptEmailParameterDeterminerTests: XCTestCase {
+    private var stores: MockStoresManager!
+
+    override func setUp() {
+        super.setUp()
+
+        stores = MockStoresManager(sessionManager: SessionManager.makeForTesting())
+        stores.whenReceivingAction(ofType: SystemStatusAction.self) { action in
+            guard case let .synchronizeSystemPlugins(_, onCompletion) = action else {
+                return
+            }
+
+            onCompletion(.success(()))
+        }
+    }
+
+    override func tearDown() {
+        stores = nil
+    }
+
+    func test_when_only_WCPay_is_active_and_version_is_higher_than_minimum_that_sends_email_then_returns_nil() {
+        let order = Order.fake()
+        let wcPayPlugin = SystemPlugin.fake().copy(version: "4.3.4")
+        let paymentsPluginsDataProvider = MockPaymentsPluginsDataProvider(wcPayPlugin: wcPayPlugin,
+                                                                      bothPluginsInstalledAndActive: false,
+                                                                      wcPayInstalledAndActive: true)
+        let sut = PaymentReceiptEmailParameterDeterminer(paymentsPluginsDataProvider: paymentsPluginsDataProvider, stores: stores)
+
+        let result: Result<String?, Error> = waitFor { promise in
```

Comment:
nit: from the comment above
```suggestion
        // When
        let result: Result<String?, Error> = waitFor { promise in
```

Reply:
👌